examples/kitchen-sink/src/App.tsx

```diff
@@ -35,7 +35,7 @@ const LargeStyle: CSSProperties = {
 }
 
 // From https://www.smashingmagazine.com/2013/07/simple-responsive-images-with-css-background-images/
-const Img: React.SFC<
+const Img: React.FunctionComponent<
```

Comment:
(non-blocking) -- note you can also use the abbreviated version, `React.FC`

Reply:
Yeah, hesitated for a bit, but I find abbreviation confusing in code (interpretation differ from peoples), which is why I choose this way. Thanks for the feedback :D